src/krylov/lobpcg.c

```diff
@@ -146,15 +146,15 @@ lobpcg_MultiVectorImplicitQR(
 }
 
 static void
-lobpcg_sqrtVector( HYPRE_Int n, HYPRE_Int* mask, HYPRE_Real* v )
+lobpcg_hypre_sqrtVector( HYPRE_Int n, HYPRE_Int* mask, HYPRE_Real* v )
```

Comment:
Here's another unnecessary name change similar to what Ulrike pointed out elsewhere.

Reply:
I thought @oseikuffuor1  wanted the change, but it does seem unnecessary to me too.

Reply:
Thanks @waynemitchell, it was a well-hidden change I missed. Thanks for catching it.